src/main/java/service/BaseballService.java

```diff
@@ -0,0 +1,60 @@
+package service;
+
+import domain.Computer;
+import domain.Result;
+import domain.User;
+
+import java.util.Random;
+
+public class BaseballService {
+    public void setRandomNumber(Computer computer) {
+        Random random = new Random();
+
+        while (true) {
+            computer.makeRandomNumberToString(random.nextInt(900) + 100);
+            if (computer.notExistsDuplicateDigit())
+                break;
+        }
+        System.out.println("computerNum : " + computer.getComputerNum());
+    }
+
+    public Result getGameResult(Computer computer, User user) {
+        Result gameResult = calculateGameResult(computer, user);
+        return gameResult;
+    }
+
+    private Result calculateGameResult(Computer computer, User user) {
+        String userNum = user.getUserNum();
+        String computerNum = computer.getComputerNum();
+        Result gameResult = new Result();
+
+        for (int i=0; i<userNum.length(); i++) {
```

Comment:
```suggestion
        for (int i = 0; i < userNum.length(); i++) {
```

java 컨벤션 위반! 마찬가지로 IDE의 자동 포매팅을 활용하는 습관을 가지면 좋을듯 해요!